src/bin/nydus-image/main.rs

```diff
@@ -399,7 +398,7 @@ fn prepare_cmd_args(bti_string: &'static str) -> App {
                     )
                     .arg(arg_output_json.clone())
             )
-    );
+                );
```

Comment:
Can't believe that cargo fmt didn't find this...

Reply:
It seems `cargo fmt` is broken from L164. 😂